e2e-tests/contentful/snapshots.js

```diff
@@ -63,13 +63,13 @@ module.exports = {
     },
     "text: Long Markdown Complex": {
-      "1": "<div data-cy-id=\"long-markdown-complex\">\n  <h1><a href=\"https://www.gatsbyjs.com/\">Headline Complex</a></h1>\n  <p>The <em>European languages</em> are members of the same family. Their\n    separate existence is a myth. For:</p>\n  <ul>\n    <li>science</li>\n    <li>music</li>\n    <li>sport</li>\n    <li>etc</li>\n  </ul>\n  <p>Europe uses the same <kbd>vocabulary</kbd>.</p>\n  <blockquote>\n    <p>The languages only differ in their grammar, their pronunciation and their\n      most common words. Everyone realizes why a new common language would be\n      desirable: one could refuse to pay expensive translators.</p>\n  </blockquote>\n  <p><img\n      src=\"//images.ctfassets.net/k8iqpp6u0ior/2PeXF5HW2hiDfCggU9RIpO/8236da1f40af17e19922ddb87030735a/jae-park-7GX5aICb5i4-unsplash.jpg\"\n      alt=\"Example Cat\"></p>\n  <p>To achieve this, it would be necessary to have uniform grammar,\n    pronunciation and more common words. If several languages coalesce, the\n    grammar of the resulting language is more simple and regular than that of\n    the individual languages. The new common language will be more simple and\n    regular than the existing European languages. It will be as simple as\n    Occidental; in fact, it will be.</p>\n  <pre><code class=\"language-json\">{\n    \"userId\": 1,\n    \"id\": 1,\n    \"title\": \"delectus aut autem\",\n    \"completed\": false\n}\n</code></pre>\n</div>"
+      "1": "<div data-cy-id=\"long-markdown-complex\">\n  <h1><a href=\"https://www.gatsbyjs.com/\">Headline Complex</a></h1>\n  <p>The <em>European languages</em> are members of the same family. Their\n    separate existence is a myth. For:</p>\n  <ul>\n    <li>science</li>\n    <li>music</li>\n    <li>sport</li>\n    <li>etc</li>\n  </ul>\n  <p>Europe uses the same <kbd>vocabulary</kbd>.</p>\n  <blockquote>\n    <p>The languages only differ in their grammar, their pronunciation and their\n      most common words. Everyone realizes why a new common language would be\n      desirable: one could refuse to pay expensive translators.</p>\n  </blockquote>\n  <p><img\n      src=\"//images.ctfassets.net/k8iqpp6u0ior/2PeXF5HW2hiDfCggU9RIpO/8236da1f40af17e19922ddb87030735a/jae-park-7GX5aICb5i4-unsplash.jpg\"\n      alt=\"Example Cat\"></p>\n  <p>To achieve this, it would be necessary to have uniform grammar,\n    pronunciation and more common words. If several languages coalesce, the\n    grammar of the resulting language is more simple and regular than that of\n    the individual languages. The new common language will be more simple and\n    regular than the existing European languages. It will be as simple as\n    Occidental; in fact, it will be.</p>\n  <pre><code class=\"language-json\">{\n\t\"userId\": 1,\n\t\"id\": 1,\n\t\"title\": \"delectus aut autem\",\n\t\"completed\": false\n}\n</code></pre>\n</div>"
```

Comment:
1. here are real tabs replaced with encoded tabs (`	` -> `\t`). Look at the end next to `<pre>`
2. the `<kbd>` tag is not touched here, this only happens in #31007

Reply:
The `<kdb>` tag is what originally raised this PR as that was the moment I figured out the snapshots are out of date.

Reply:
Tabs in pre is an expected remark chang. The full list of tree changes is here: https://github.com/remarkjs/remark/releases/tag/13.0.0 I think tabs are listed as `fix: tabs` 😄

Reply:
Super!